lib/arrayOrder.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Takes an array, and orders the numbers from least to greatest.
+ * 
+ * @param {Array} An array of numbers.
+ *
+ * @return {Array} An array of numbers ordered from least to greatest.
+ * 
+ */
+function arrayOrder(arr) {
```

Comment:
Can we call this `sort`?

Reply:
sortArray or arraySort?

I'm cool with either.

Reply:
I was think just `sort`. Is there anything else we can sort?

Reply:
Cool. I will go ahead and make those updates.

Reply:
Awesome! This is totally up for discussion too. If you had a different idea feel free to tell me to back off. Thanks and Happy Hacktoberfest 🎃 !

Reply:
I'm stuck on the test. Not sure how I would test with accepting a function, because the result would depend on the sorting function that was entered.

Sorry, I am new to writing tests.

Reply:
Never mind, I think I know what to do.